dropin-checkout/client/react/inai-drop-in/src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import DropIn from './component/Dropin';
+import Payment from './component/Payment';
+import Product from './component/Product';
+
+const App = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route exact path='/' element={<DropIn />} />
+        <Route exact path='/product' element={<Product />} />
+        <Route exact path='/payment' element={<Payment />} />
```

Comment:
Can we update the route to checkout and call the component Checkout as well?
```suggestion
        <Route exact path='/checkout' element={<Checkout />} />
```